common/src/main/java/com/sequenceiq/cloudbreak/auth/crn/Crn.java

```diff
@@ -352,6 +352,7 @@ public static Region safeFromString(String input) {
         BLUEPRINT("blueprint"),
         CLUSTER_DEFINITION("clusterdefinition"),
         CLUSTER_TEMPLATE("clustertemplate"),
+        CUSTOM_CONFIGS("customconfigs"),
```

Comment:
how about customconfiguration?

Reply:
have you updated related Crn class in thunderhead codebase?

Reply:
please update this based on your PR in thunderhead https://github.infra.cloudera.com/thunderhead/thunderhead/pull/5613